app/components/GettingStartedModal/steps/SecretsStep.tsx

```diff
@@ -38,11 +43,10 @@ const SecretsStep = ({ selectedEnvironment, typedSecrets, onChange }: Props) =>
                     onChange(secretInputs);
                 };
                 return (
-                    <Form.Field key={name}>
+                    <Form.Field key={name} label={label} help={hasSecretDescrtiption ? secretDescription : undefined}>
```

Comment:
I think we shouldn't add "Secret description" to all secrets. We just need to have the framework ready to support new property.

Regarding change in that line, I'm wondering if we change L41 to:
```
            {selectedEnvironment.secrets.map(({ name, label, type, description }) => {
```
could we have here just:
```suggestion
                    <Form.Field key={name} label={label} help={description}>
```
? 🤔

Reply:
https://github.com/cloudify-cosmo/cloudify-stage/pull/2160/commits/53662860fd59f312c20c62992405bb3b63c24b0f applied here :)